mobile/src/main/res/values/strings.xml

```diff
@@ -38,11 +38,11 @@
     <string name="settings_openhab_sslclientcert_howto">SSL client certificate how-to</string>
     <string name="settings_openhab_sslclientcert_howto_url" translatable="false">https://github.com/openhab/openhab.android/blob/master/KeyAuth.md</string>
     <string name="settings_openhab_screentimeroff">Disable display timer</string>
-    <string name="settings_openhab_screentimeroff_summary">Disable display switch off timer when HABDroid is running</string>
+    <string name="settings_openhab_screentimeroff_summary">Disable display switch off timer when openHAB is running</string>
     <string name="settings_openhab_demomode">Demo mode</string>
-    <string name="settings_openhab_demomode_summary">Run HABDroid in demo mode</string>
-    <string name="settings_openhab_theme">HABDroid Theme</string>
-    <string name="settings_openhab_appversion">HABDroid Version</string>
+    <string name="settings_openhab_demomode_summary">Run openHAB App in demo mode</string>
```

Comment:
Sorry for bringing this up at the last minute, but when testing this "Run openHAB App" sounds broken.  This should  probably  "Run in demo mode"